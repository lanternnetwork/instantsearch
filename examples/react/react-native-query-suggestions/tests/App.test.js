import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

Date.now = jest.fn(() => 0);

test('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
