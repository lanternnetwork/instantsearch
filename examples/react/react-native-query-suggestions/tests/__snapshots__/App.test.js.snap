// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "marginTop": 30,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "flexDirection": "row",
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "https://www.algolia.com/static_assets/images/press/downloads/algolia-mark-blue.png",
        }
      }
      style={
        Object {
          "height": 40,
          "margin": 10,
          "width": 40,
        }
      }
    />
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      autoCorrect={false}
      clearButtonMode="always"
      onChange={[Function]}
      onChangeText={[Function]}
      onFocus={[Function]}
      placeholder="Search a product..."
      placeholderTextColor="black"
      rejectResponderTermination={true}
      spellCheck={false}
      style={
        Object {
          "alignSelf": "center",
          "color": "black",
          "height": 50,
          "width": 300,
        }
      }
      underlineColorAndroid="white"
      value=""
    />
  </View>
  <Text
    style={
      Object {
        "backgroundColor": "lightgrey",
        "height": 40,
        "justifyContent": "center",
        "padding": 10,
      }
    }
  >
    Best results
  </Text>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      ItemSeparatorComponent={[Function]}
      data={Array []}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      numColumns={1}
      onContentSizeChange={[Function]}
      onEndReached={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;
